setup.py

```diff
@@ -246,6 +246,14 @@ def configuration(parent_package='', top_path=None):
         extras_require=extras_require,
         python_requires='>=3.8',
         packages=setuptools.find_packages(exclude=['doc', 'benchmarks']),
+        package_data={
+            # distribute Cython source files in the wheel
+            "": ["*.pyx", "*.pxd", "*.pxi", ""],
+            # distribute data required by the tests
+            "color": ["tests/data/*.npy"],
```

Comment:
Are you sure we need the test data? It can be added via pooch which reduces the size of the package we ship

Reply:
https://github.com/scikit-image/scikit-image/blob/852e9daccf6c394a96fd662ae1f1c5488f1b6596/skimage/data/_registry.py#L48

Reply:
Right, I forgot about that. Indeed the test cases all use `skimage._shared.testing.fetch` when referring to those files, so it should just work as-is.

Overall the total of the `color` and `registration` test data is < 500kB uncompressed, so it is not too large in any case, but better to avoid it if we can!